### PR TITLE
move tokenizer model to third party llama2

### PR DESCRIPTION
**Pull request by FanhaiLu1**
Move tokenizer model to third party llama2

### CODE REVIEW

File: jetstream/tests/engine/test_token_utils.py [hunk 1]
```diff
@@ -48,7 +48,7 @@ def decode(self, t: int) -> str:
 
 class TokenUtilsTest(unittest.TestCase):
     def setup(self):
-        tokenizer_path = "tokenizer.model"
+        tokenizer_path = "third_party/llama2/tokenizer.model"
         current_dir = os.path.dirname(__file__)
         tokenizer_path = os.path.join(current_dir, tokenizer_path)
         print(f"model_path: {tokenizer_path}")
```
Comment (patemotter) [inline jetstream/tests/engine/test_token_utils.py:54]: Nit outside of this change: Should probably be renamed to tokenizer_path. 

Also typos in comments above "Tokenier" -> "Tokenizer"

**Review by patemotter — APPROVED** [review]
Overall LGTM. Would some of these tests break if we were to change the tokenizer being used?

File: jetstream/tests/engine/test_token_utils.py (hunk 1 above)
Comment (FanhaiLu1) [reply jetstream/tests/engine/test_token_utils.py:54]: Thanks Pate! there could be more data set from llama2. Keep it as "third_party/llama2"

**Comment by FanhaiLu1** [comment]
> Overall LGTM. Would some of these tests break if we were to change the tokenizer being used?

The test_underscore_in_output test is specific for a  under score token id, it may fail if tokenizer changed. All other 3 test are  tokenizer independent

**Pull request merged**